src/vs/platform/issue/electron-main/issueMainService.ts

```diff
@@ -175,6 +175,15 @@ export class IssueMainService implements ICommonIssueService {
 				event.sender.send('vscode:windowsInfoResponse', info.windows);
 			});
 		});
+
+		ipcMain.on('vscode:killProcess', (_: unknown, arg: { pid: number, force: boolean }) => {
```

Comment:
I would like to reduce raw IPC usage, so please introduce a method for this in the `IElectronService`

Reply:
Added!